### PR TITLE
AISDK-221 Fix HIPAA Error Message on Incorrect Usage

### PR DESCRIPTION
**Pull request by aaron-wilson-rev**
The python sdk does not try to wrap any of the http error responses from the API, so no need to fix handling of a 403 here. This PR just removes some unnecessary test entries.

### CODE REVIEW

File: tests/helpers/errors.py [hunk 1]
```diff
@@ -3,16 +3,6 @@ def get_error_test_cases(errors):
 
 
 ERRORS = {
-    'invalid-parameters': {
```
Comment (dmtrrk) [inline tests/helpers/errors.py:6]: why did you remove this?

Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:6]: It's not used

File: tests/helpers/errors.py [hunk 2]
```diff
@@ -22,13 +12,6 @@ def get_error_test_cases(errors):
         "title": "could not find job",
         "status": 404
     },
-    'out-of-credit': {
-        "title": "You do not have enough credits",
-        "type": "https://www.rev.ai/api/v1/errors/out-of-credit",
-        "detail": "You have only 60 seconds remaining",
-        "current_balance": 60,
-        "status": 403
-    },
```
Comment (eugenep-rev) [inline tests/helpers/errors.py:31]: can you explain the fix? why just removing fixes the issue?

Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:31]: there isn't actually a problem with the python sdk since it just passes through the api error. I just removed these from our test resources since they are never used in our tests.

File: tests/helpers/errors.py (hunk 1 above)
Comment (dmtrrk) [reply tests/helpers/errors.py:6]: I think, we might need to cleanup [tests](https://github.com/revdotcom/revai-python-sdk/blob/develop/tests/test_baseclient.py#L31) if we are deleting test cases

File: tests/helpers/errors.py [hunk 3]
```diff
@@ -4,31 +4,23 @@ def get_error_test_cases(errors):
 
 ERRORS = {
     'invalid-parameters': {
-        "parameter": {
-            "example": [
-                "The example field is required"
-            ]
-        },
         "type": "https://www.rev.ai/api/v1/errors/invalid-parameters",
-        "title": "Your request parameters didn't validate",
```
Comment (aaron-wilson-rev) [inline tests/helpers/errors.py:13]: this breaks regex on the test verification so easiest to just cut it out. I'm guessing this is why it was left out

File: tests/helpers/errors.py (hunk 1 above)
Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:6]: this error wasn't actually being tested before, so instead of removing it I fixed it and included it in the tests to make sure we raise a 400 response properly

File: tests/helpers/errors.py [hunk 4]
```diff
@@ -4,31 +4,23 @@ def get_error_test_cases(errors):
...
         "status": 400
     },
     'unauthorized': {
         "title": "Authorization has been denied for this request",
         "status": 401
     },
+    'forbidden': {
+        "status": 403,
+        "error": "HIPAA user is not allowed to use MediaUrl"
+    },
```
Comment (eugenep-rev) [inline tests/helpers/errors.py:18]: I"m confused about this PR. Looks like we change tests only here. Also, we return 403 when we use wrong language or human transcriber.

Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:18]: Yes, because we have test cases that today are not used in the test and are misleading. I'll make the 403 test more generic. The main point of this test is just that we raise the error if we get one back from the api.

File: tests/helpers/errors.py (hunk 1 above)
Comment (dmtrrk) [reply tests/helpers/errors.py:6]: correct me if I'm wrong, but in the test you're trying to get a test data by the provided key in `@pytest.mark.parametrize`. and once you deleted test data, this function might return `None` and it might be the test is not failing eventually

Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:6]: 'invalid-parameters' was not included in the parameters before

Comment (aaron-wilson-rev) [reply tests/helpers/errors.py:6]: But now I added it to the parameters and restored it in the errors file so we can test it too.

**Review by dmtrrk — APPROVED** [review]
